app/Models/ReportbackItem.php

```diff
@@ -28,4 +28,12 @@ public function reactions()
     {
         return $this->hasMany('\Rogue\Models\Reaction');
     }
+
+    /**
+     * Each reportback item has one review.
+     */
+    public function review()
```

Comment:
@DFurnes yea they are. ReportbackItem was from the old DB structure and `Post` is the new. At one point we had reviews coming in from both so we had to continue to support both structures which is one of the reasons we turned off sending to rogue so that we could just get all of the reportbacks into rogue as a `Post`. So I don't think this PR needs to support `ReportbackItem`s anymore @chloealee